src/internal/SchemaBase.ts

```diff
@@ -156,6 +159,22 @@ export const Lazy: S2.Schemable2<URI>['lazy'] = (id, f) =>
  */
 export const Readonly: S2.Schemable2<URI>['readonly'] = identity
 
+/**
+ * @since 1.0.0
+ * @category Combinators
+ */
+export const Brand =
```

Comment:
don't you have a linter rule set up to require explicit types on exports?

Reply:
No, does that affect the export (import?) mappings?  If so, we should definitely add that.  

Here, I wasn't able to annotate these explicitly even though the types were identical.  Not sure why 🤔 The error was something like two incompatible generics, A not assignable to A